Bottom: 6ff4c3fef859b775522c27db8de4a8fcfcf792c3
Top:    1ab9fd04718504b5cc665dea14e9fa9df0298d6c
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2013-02-01 22:37:01 -0800

Refresh of kern-bug

---

diff --git a/src/cma.c b/src/cma.c
index 547781e..06bea44 100755
--- a/src/cma.c
+++ b/src/cma.c
@@ -1954,6 +1954,7 @@ retry:
 	if (resp.uid) {
 		evt->id_priv = (void *) (uintptr_t) resp.uid;
 	} else {
+printf("no uid event %s\n", rdma_event_str(resp.event));
 		evt->id_priv = ucma_lookup_id(resp.id);
 		if (!evt->id_priv) {
 			fprintf(stderr, PFX "Warning: discarding unmatched "
